src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 40 * days;
+
+  if (days >= 7) {
+    totalAmount -= 50;
+
+    return totalAmount;
+  }
+
+  if (days >= 3) {
```

Comment:
Consider using a constant variable for the discount amount (20) for rentals of 3 days or more for better readability and maintainability.